sdk/js/src/solana/sendAndConfirmPostVaa.ts

```diff
@@ -40,22 +40,32 @@ export async function postVaaWithRetry(
 
   const postVaaTransaction = unsignedTransactions.pop()!;
 
-  const responses = await sendAndConfirmTransactionsWithRetry(
-    connection,
-    modifySignTransaction(signTransaction, ...signers),
-    payer.toString(),
-    unsignedTransactions,
-    maxRetries
-  );
-  //While the signature_set is used to create the final instruction, it doesn't need to sign it.
+  const options: ConfirmOptions = {
+    commitment,
+    maxRetries,
+  };
+
+  const responses: TransactionSignatureAndResponse[] = [];
+  for (const transaction of unsignedTransactions) {
+    const response = await signSendAndConfirmTransaction(
```

Comment:
should we be stacking these promises instead of waiting for each?

Reply:
It's deprecated now so fine either way. The `postVaa` function has an Async parameter for stacking the promises